rule.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto2";
+package dv360_excel_plugin;
+
+// A message that represents a scheduled rule for manipulating DV360 entities.
+message Rule {
+  // required
+  // The name of the rule, set by the user.
+  optional string name = 1;
+
+  // required
+  // The id that uniquely identified each rule.
+  optional int32 id = 2;
+
+  // required
+  optional Action action = 3;
+
+  // required
+  optional Schedule schedule = 4;
+}
+
+// A message representing the action applied to a DV360 entity.
+message Action {
+  enum Type {
+    UNSPECIFIED_TYPE = 0;
+    CHANGE_LINE_ITEM_STATUS = 1;
+  }
+
+  // required
+  // The type of action.
+  optional Type type = 1;
+
+  // required
+  // The parameters used for each type of action.
+  oneof params {
+    ChangeLineItemStatusParams change_line_item_status_params = 2;
+  }
+}
+
+// The action parameters to change line items' status.
+message ChangeLineItemStatusParams {
+  // required
+  // The ID of line items to which the action applies.
+  repeated int64 line_item_ids = 1;
+
+  enum Status {
+    UNSPECIFIED_STATUS = 0;
+    ACTIVE = 1;
+    PAUSED = 2;
+  }
+
+  // required
+  // The status of which the line item will be changed to.
+  // Can be set to either active or paused.
```

Comment:
Is it necessary if the Enum above already shows that?

Reply:
Sorry, which Enum?

Reply:
Status

Reply:
Oh gotcha, I'll remove it.